Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -175,5 +141,52 @@ bool IsSuppliedPluginFolderName()
             }
         }
     }
+
+    public class PluginFinder<TPlugin> 
+    {
+        private static readonly ILog _logger = LogManager.GetLogger(typeof(PluginFinder<>));
```

Comment:
thoughts about using System.Reflection.MethodBase.GetDeclaringMethod() instead of the typeof statement?

Reply:
This method does not exists on the MethodBase class.